include/mqtt/broker/session_state.hpp

```diff
@@ -179,48 +222,21 @@ struct session_state {
         v5::properties props) {
 
         if (online()) {
-            // TODO: Probably this should be switched to async_publish?
-
-            //       Given the async_client / sync_client seperation
-            //       and the way they have different function names,
-            //       it wouldn't be possible for broker.hpp to be
-            //       used with some hypothetical "async_server" in the future.
-            con_->publish(
+            publish(
+                ioc,
```

Comment:
Why remove comment about async ? I was wondering if this could be an async operation or not ? Is there some reason why this is not async ?

Reply:
I don't remove it. It moved to publish()